redirect.go

```diff
@@ -7,5 +7,10 @@ import (
 
 func main() {
 	dest := os.Getenv("DESTINATION")
-	fmt.Printf("status: 302\nlocation: %s\n\n", dest)
+	status := os.Getenv("STATUS")
```

Comment:
Should we require a relevant status, or at least warn+require an override flag if someone is using an "unexpected" redirect status?

Reply:
Enforcing valid status codes makes sense to me. 

Just to be sure, in our case the codes would be 

Permanent redirect - 301, 308
Temporary redirect - 302, 303, 307

Correct me if I'm wrong but the special redirects don't apply to this particular use case. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections